server/modules/arenaImport/service/arenaImport/model/arenaSurveyFileZip.js

```diff
@@ -69,9 +69,8 @@ export const getUsers = async (streamZipFile) => {
   return JSON.parse(users)
 }
 export const getUser = async (streamZipFile, uuid) => {
-  const userPath = getPathByUuid(uuid)(getPaths(streamZipFile))
-  const user = await streamZipFile.getEntryData(userPath)
-  return JSON.parse(user)
+  const users = await getUsers(streamZipFile)
+  return (users || []).find((user) => user.uuid === uuid)
```

Comment:
this method can be remove because in 
_server/modules/arenaImport/service/arenaImport/metaImportJobs/usersImportJob.js_
you already have the list of users at line 86, so lines 88-90 can be removed as well

Reply:
Changed